src/ResourceManagement/Resource/ResourceManagement/Generated/Models/DeploymentDebugDetailLevel.cs

```diff
@@ -0,0 +1,40 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Management.Resources.Models
+{
+    /// <summary>
+    /// Specifies the deployment debug detail level for the deployment
+    /// operations.
+    /// </summary>
+    [Flags]
+    public enum DeploymentDebugDetailLevel
```

Comment:
IMO, we call it DeploymentDebugLogLevel to keep it consistent across clients. Also, "All" is a valid option when someone wants to specify both request and response log levels.